app/src/main/java/org/p2p/wallet/home/MainFragment.kt

```diff
@@ -105,6 +110,17 @@ class MainFragment :
             onCreateActions = arrayListOf()
         }
 
+        deeplinksManager
+            .subscribeOnDeeplinks(
```

Comment:
please place subscribeOnDeeplinks on one line with deeplinksManager. we have a formatting rule for that in styleguide i think